src/components/CanvasWrapper.tsx

```diff
@@ -422,27 +421,24 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
 
   const onMouseMove = useCallback(
     (event: MouseEvent): void => {
-      const isMouseLeftHeldWithModifier = isMouseLeftDown.current && (event.ctrlKey || event.metaKey || enablePan);
-      if (isMouseLeftHeldWithModifier || isMouseMiddleDown.current || isMouseRightDown.current) {
-        canv.domElement.style.cursor = "grabbing";
-        handlePan(event.movementX, event.movementY);
+      if (isMouseLeftDown.current || isMouseMiddleDown.current || isMouseRightDown.current) {
         // Add to total drag distance; if it exceeds threshold, consider the mouse interaction
-        // to be a drag and disable track selection.
+        // to be a drag operation. Start panning and disable track selection.
         totalMouseDrag.current.x += Math.abs(event.movementX);
         totalMouseDrag.current.y += Math.abs(event.movementY);
-        if (!isMouseDragging.current && totalMouseDrag.current.length() > MIN_DRAG_THRESHOLD_PX) {
+        if (totalMouseDrag.current.length() > MIN_DRAG_THRESHOLD_PX) {
           isMouseDragging.current = true;
+          handlePan(event.movementX, event.movementY);
         }
+      }
```

Comment:
We only start panning once the mouse has moved more than a total of `MIN_DRAG_THRESHOLD_PX`. If a user clicks and their mouse moves slightly, this means that the canvas won't immediately pan.